projects/igniteui-angular/src/lib/directives/dragdrop/dragdrop.directive.ts

```diff
@@ -416,11 +416,6 @@ export class IgxDragDirective implements OnInit, OnDestroy {
     ngOnDestroy() {
         this._destroy.next(true);
         this._destroy.unsubscribe();
-
-        if (this._dragGhost) {
```

Comment:
@skrustev, I can reproduce it only in the case with multi column headers, otherwise the the drag Ghost is not removed while scrolling. I can investigate that further, what do you think?